CMSgov/cmcs-eregulations
# EREGCSC-1656 -- Add links to GovInfo historical CFR pages

Resolves [EREGCSC-1656](https://jiraent.cms.gov/browse/EREGCSC-1656)

**Description:**

Front end work for GovInfo Historical Pages feature.

**This pull request changes:**

- Refactors existing Copy Button Tooltip to be more generic Tooltip component
- Creates Regulation History tooltip content, populated via API call to history endpoint
- Styles everything to match [Figma Comp](https://www.figma.com/file/d4iODwrdAEMS3e45LiWi2K/Timeline-with-GovInfo-Links?node-id=1869%3A429&t=sAzCcXWoIpWrSfRr-0)

**Steps to manually verify this change:**

1. Visit site and go to any section in the reader view (example: [440.10](https://a01kqidbx9.execute-api.us-east-1.amazonaws.com/dev677/42/440/Subpart-A/2020-12-16/#440-10))
2. click "View § 440.10 Regulation History"
3. Make sure works as expected and matches [Figma comp](https://www.figma.com/file/d4iODwrdAEMS3e45LiWi2K/Timeline-with-GovInfo-Links?node-id=1869%3A429&t=sAzCcXWoIpWrSfRr-0)
